src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellSectionRootRenderer.cs

```diff
@@ -568,6 +568,17 @@ void LayoutHeader()
 				CGRect frame = new CGRect(View.Bounds.X, headerTop, View.Bounds.Width, HeaderHeight);
 				_blurView.Frame = frame;
 				_header.ViewController.View.Frame = frame;
+#if MACCATALYST
```

Comment:
On iOS, it's not necessary, right? If, for example, the device changes orientation.